home/pi/auto_run.sh

```diff
@@ -302,24 +299,29 @@ function setup_wizard() {
              1)
                 echo "$key - Analog audio"
                 # audio out the analog speaker/headphone jack
-                sudo amixer cset numid=3 "1" > /dev/null 2>&1
-                echo 'sudo amixer cset numid=3 "1" > /dev/null 2>&1' >> ~/audio_setup.sh
+                pactl set-default-sink alsa_output.platform-bcm2835_audio.analog-stereo
+                echo 'pactl set-default-sink alsa_output.platform-bcm2835_audio.analog-stereo' >> ~/audio_setup.sh
                 audio="analog_audio"
                 break
                 ;;
              2)
                 echo "$key - HDMI audio"
                 # audio out the HDMI port (e.g. TV speakers)
-                sudo amixer cset numid=3 "2" > /dev/null 2>&1
-                echo 'sudo amixer cset numid=3 "2"  > /dev/null 2>&1' >> ~/audio_setup.sh
+                pactl set-default-sink alsa_output.platform-bcm2835_audio.digital-stereo
+                echo 'pactl set-default-sink alsa_output.platform-bcm2835_audio.digital-stereo' >> ~/audio_setup.sh
                 audio="hdmi_audio"
                 break
                 ;;
              3)
                 echo "$key - USB audio"
                 # audio out to the USB soundcard
-                sudo amixer cset numid=3 "0" > /dev/null 2>&1
-                echo 'sudo amixer cset numid=3 "0"  > /dev/null 2>&1' >> ~/audio_setup.sh
+                echo "Select your output device"
+                pactl list sinks short | awk '{printf("  %d) %s\n", NR, $2)}'
+                echo -n "${HIGHLIGHT}Choice:${RESET} "
+                read -N1 -s card_num
+                card_name=$(pactl list sinks short | awk '{print$2}' | sed -n ${card_num}p)
+                pactl set-default-sink ${card_name}
+                echo 'pactl set-default-sink ${card_name}' >> ~/audio_setup.sh
```

Comment:
I kind of wonder if it would behoove us here to emit ``pactl unload-module module-suspend-on-idle`` by default here, given that we know that suspension is a problem with many devices.  If that doesn't sound reasonable, maybe we could add it as a comment to the sample audio_setup.sh above?